lib/Gateway.js

```diff
@@ -9,7 +9,7 @@ const path = require('path')
 const reqlib = require('app-root-path').require
 const utils = reqlib('/lib/utils.js')
 const EventEmitter = require('events')
-const { AlarmSensorType, BinarySensorType } = require('zwave-js')
+const AlarmSensorType = require('zwave-js')
```

Comment:
That (default) import is not correct.
```suggestion
const { AlarmSensorType } = require('zwave-js')
```